packages/graphile-build-pg/src/plugins/PgAllRows.js

```diff
@@ -103,6 +107,24 @@ export default (async function PgAllRows(
                                 builder.setOrderIsUnique();
                               }
                             });
+                          } else if (isView(table) && !!uniqueIdAttribute) {
+                            builder.beforeLock("orderBy", () => {
+                              builder.data.cursorPrefix = [
+                                "view_unique_key_asc",
+                              ];
+                              builder.orderBy(
+                                sql.fragment`${builder.getTableAlias()}.${sql.identifier(
+                                  uniqueIdAttribute.name
+                                )}`,
+                                true
+                              );
+                              builder.setOrderIsUnique();
+                            });
+                          } else {
+                            builder.orderBy(
+                              sql.fragment`(row_number() over (partition by 1))`,
+                              true
+                            );
```

Comment:
I'm going to remove this `else` clause so we don't change the existing cursor format.

Reply:
No I'm not... But I'm going to tweak it slightly.